### PR TITLE
feat: update wallet notification with npg

### PR DESCRIPTION
**Pull request by infantesimone**
### Motivation and context

The goal of this PR is to update wallet notifications with NPG specifications instead HPP specifications.

### Type of changes

- [ ] Add new resources
- [ ] Update configuration to existing resources
- [ ] Remove existing resources

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### CODE REVIEW

**Review by Fernando-Granato — APPROVED** [review]

File: src/domains/wallet-app/api/npg-notifications/v1/_openapi.json.tpl [hunk 1]
```diff
@@ -0,0 +1,594 @@
...
+    "description": "Nexi API to handle onboarding",
+    "url": "https://developer.nexi.it/it/api/notifica"
+  },
+  "servers": [
+    {
+      "url": "https://${hostname}"
+    }
+  ],
+  "paths": {
+    "/wallets/{walletId}/sessions/{orderId}": {
```
Comment (Fernando-Granato) [inline src/domains/wallet-app/api/npg-notifications/v1/_openapi.json.tpl:31]:
```suggestion
    "/wallets/{walletId}/sessions/{orderId}/notifications": {
```

File: src/domains/wallet-app/api/npg-notifications/v1/_openapi.json.tpl [hunk 2]
```diff
@@ -0,0 +1,594 @@
...
+      "url": "https://${hostname}"
+    }
+  ],
+  "paths": {
+    "/wallets/{walletId}/sessions/{orderId}/notifications": {
+      "post": {
+        "tags": [
+          "walletNotifications"
+        ],
+        "summary": "Update Wallet on NPG payment instrument creation",
```
Comment (Fernando-Granato) [inline src/domains/wallet-app/api/npg-notifications/v1/_openapi.json.tpl:36]:
```suggestion
        "summary": "Update Wallet on NPG onboarding authorization response",
```

File: src/domains/wallet-app/api/npg-notifications/v1/_openapi.json.tpl [hunk 3]
```diff
@@ -0,0 +1,594 @@
...
+        "description": "Update Wallet on NPG payment instrument creation",
```
Comment (Fernando-Granato) [inline src/domains/wallet-app/api/npg-notifications/v1/_openapi.json.tpl:37]:
```suggestion
        "description": "Update Wallet on NPG onboarding authorization response",
```

File: src/domains/wallet-app/api/npg-notifications/v1/_openapi.json.tpl (hunk 1 above)
Comment (infantesimone) [reply src/domains/wallet-app/api/npg-notifications/v1/_openapi.json.tpl]: done :-)

**Review by Fernando-Granato — APPROVED** [review]

**Pull request merged**